.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
how does a person create an issue with a proposed enhancement? how do we create issues for the roadmap? epics?

it looks like right now based on these changes we would only be able to report bugs in code or docs, and nothing else

Reply:
I had the same concern above, so I'm re-enabling blank issues.